nomad/local/grapl-local-infra.nomad

```diff
@@ -527,6 +541,50 @@ job "grapl-local-infra" {
     }
   }
 
+  group "uid-allocator-db" {
```

Comment:
the more we do this the more convinced I am that "we probably don't need 4 concurrent instances/clusters of postgres" but whatever